src/gps_agent_pkg/src/robotplugin.cpp

```diff
@@ -532,3 +548,38 @@ void RobotPlugin::get_fk_solver(boost::shared_ptr<KDL::ChainFkSolverPos> &fk_sol
         ROS_ERROR("Unknown ArmType %i requested for joint encoder readings!",arm);
     }
 }
+
+
+
```

Comment:
nitpick - can you remove all this space? should be 1 line of whitespace between functions.